snap/pack/pack.go

```diff
@@ -117,15 +117,16 @@ func loadAndValidate(sourceDir string) (*snap.Info, error) {
 	if err != nil {
 		return nil, err
 	}
+	snap.AddImplicitHooksFromContainer(info, container)
```

Comment:
I attempted approach to create /internal helpers and share code with ReadInfoFromSnapFile, but concluded it does not look/work nice. As far as I can see not possible without exporting new function, so exporting AddImplicitHooksFromContainer seems like an ok trade-off.